### PR TITLE
Update deploy-vue-storefront-cloud.yml

### PR DESCRIPTION
**Pull request by renanoliveira0**
https://vsf.atlassian.net/browse/PL-1364

Attention, this PR adds breaking changes, it must be approved along with 2 other PRs and all must be deployed atomically, and tested immediately. This is not possible to do in advance, so it is necessary to observe who is responsible for deploying all the PRs and doing the tests.

Attention, the breaking change even breaks production, as it is the two templates that are changed, so the tests must be done in stage and then in production to validate and in case of any problem, the three PRs must be immediately reversed.

https://github.com/vuestorefront/platform-backend/pull/340
https://github.com/vuestorefront/sample-express-application-with-middleware/pull/2

### CODE REVIEW

**Review by Baroshem — APPROVED** [review]

**Review by nestak — APPROVED** [review]

**Comment by renanoliveira0** [comment]
[vsf.atlassian.net/browse/PL-1364](https://vsf.atlassian.net/browse/PL-1364)

Attention, this PR adds breaking changes, it must be approved along with 2 other PRs and all must be deployed atomically, and tested immediately. This is not possible to do in advance, so it is necessary to observe who is responsible for deploying all the PRs and doing the tests.

Attention, the breaking change even breaks production, as it is the two templates that are changed, so the tests must be done in stage and then in production to validate and in case of any problem, the three PRs must be immediately reversed.

https://github.com/vuestorefront/platform-backend/pull/340
https://github.com/vuestorefront/sample-express-application-with-middleware/pull/2

**Review by FRSgit — APPROVED** [review]

File: .github/workflows/deploy-vue-storefront-cloud.yml [hunk 1]
```diff
@@ -8,12 +8,12 @@ jobs:
...
       - name: Checkout code
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3
 
       - name: Setup node
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
-          node-version: "12.x"
+          node-version: "18.x"
```
Comment (Frodigo) [inline .github/workflows/deploy-vue-storefront-cloud.yml:16]: It was requested to change the Node version from 12 to 14, not 18. So please change it to 14 to meet the requirements.

**Review by Frodigo — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy-vue-storefront-cloud.yml (hunk 1 above)
Comment (renanoliveira0) [reply .github/workflows/deploy-vue-storefront-cloud.yml:16]: https://github.com/nodejs/Release

I figured the requirements were wrong as you can see from the link and the image.

Version 14 is at end of its life and the current LTS version is 18.

However, it's up to you to decide. Do you want me to leave it in version 14 anyway?

image

Comment (Frodigo) [reply .github/workflows/deploy-vue-storefront-cloud.yml:16]: Hi, @vuestorefront/integrations-team,

which version of Node is used in the eCommerce integrations? 

This repository is the sample Express template we used in the VSF Console when we bootstrap a new instance. 

Before this PR, there was Node 12. Should we update it to Node 14, 16, or 18? 

We want to use the Node version commonly used in the integrations.

Comment (lsliwaradioluz) [reply .github/workflows/deploy-vue-storefront-cloud.yml:16]: > which version of Node is used in the eCommerce integrations?

@Frodigo It's Node v16 or higher.

Comment (Frodigo) [reply .github/workflows/deploy-vue-storefront-cloud.yml:16]: @renanoliveira0 based on the comment above, can you change Node to Node 16?

Comment (renanoliveira0) [reply .github/workflows/deploy-vue-storefront-cloud.yml]: Done: 5e8adf2d90aa7ab3001bb74ff5489e98c1aa0fbe

**Comment by renanoliveira0** [comment]
You can merge this PR here and it won't break anything. It closes this task: https://github.com/vuestorefront/sample-express-application/pull/1

Then we discussed open PR on the Platform/Console backend.

**Review by Frodigo — APPROVED** [review]

**Pull request merged**